cortex-mixin/dashboards/writes.libsonnet

```
@@ -12,17 +12,24 @@ local utils = import 'mixin-utils/utils.libsonnet';
        })
       .addPanel(
         $.panel('Samples / s') +
-        $.statPanel('sum(cluster_namespace_job:cortex_distributor_received_samples:rate5m{%s})' % $.jobMatcher($._config.job_names.distributor), format='reqps')
+        $.statPanel(
+          'sum(%(jobAggregationPrefix)s:cortex_distributor_received_samples:rate5m{%(job)s})' % {
+            job: $.jobMatcher($._config.job_names.distributor),
+            jobAggregationPrefix: $._config.job_aggregation_prefix,
+          },
```

[nit] If I'm not missing anything, you may simplify it as:

```suggestion
          'sum(%(job_aggregation_prefix)s:cortex_distributor_received_samples:rate5m{%(job)s})' % $._config + {
            job: $.jobMatcher($._config.job_names.distributor),
          },
```